go.mod

```diff
@@ -25,7 +25,7 @@ require (
 	github.com/cyphar/filepath-securejoin v0.3.6
 	github.com/dustin/go-humanize v1.0.1
 	github.com/evanphx/json-patch v5.9.0+incompatible
-	github.com/expr-lang/expr v1.16.9
```

Comment:
Looks like the CVE was fixed in 1.17.0. Could we upgrade to that version to keep the cherry-pick minimal?